manager/volume.go

```diff
@@ -28,14 +28,18 @@ type VolumeManager struct {
 
 	currentNodeID string
 	sb            *SupportBundle
+
+	proxyConnCounter util.Counter
```

Comment:
Do we use it?

Reply:
Yes, for example, https://github.com/c3y1huang/longhorn-manager/blob/grpc-proxy-connection-metric/manager/engine.go#L38

Reply:
Thanks! I missed that.